mne/event.py

```diff
@@ -745,26 +745,37 @@ def make_fixed_length_events(raw, id, start=0, stop=None, duration=1.):
         of the recording.
     duration: float
         The duration to separate events by.
+    first_samp: bool
+        If True (default), times will have raw.first_samp added to them, as
+        in :func:`mne.find_events`. This behavior is not desirable if the 
+        returned events will be combined with event times that already 
+        have ``raw.first_samp`` added to them, e.g. event times that come 
+        from :func:`mne.find_events`.
 
     Returns
     -------
     new_events : array
         The new events.
     """
     start = raw.time_as_index(start)
-    start = start[0] + raw.first_samp
     if stop is not None:
         stop = raw.time_as_index(stop)
-        stop = min([stop[0] + raw.first_samp, raw.last_samp + 1])
     else:
-        stop = raw.last_samp + 1
+        stop = raw.last_samp + 1    
```

Comment:
``stop`` should be made a list for it to work (or unwrap the value from ``time_as_index``). That's why Travis has failures.